<launch>
  <arg name="algorithm" default="se3dif" />
  <arg name="config_file" default="" />
  <node name="grasp_planner_node_$(arg algorithm)" pkg="grasping_benchmarks_ros" type="grasp_planner_service.py" ns="$(arg algorithm)" output="screen">
    <param name="algorithm" value="$(arg algorithm)" />
    <param name="config_file" value="$(arg config_file)" />
  </node>
</launch>
